pkg/api/v1/helper/helpers.go

```diff
@@ -27,6 +27,20 @@ import (
 	"k8s.io/kubernetes/pkg/api/helper"
 )
 
+// IsExtendedResourceName returns true if the resource name is not in the
+// default namespace, or it has the opaque integer resource prefix.
+func IsExtendedResourceName(name v1.ResourceName) bool {
+	return IsCustomNamespacedResourceName(name) || IsOpaqueIntResourceName(name)
+}
+
+// Returns true if the resource name is not in the *kubernetes.io/ namespace.
+// Note: partially-qualified (unprefixed) names are assumed by default to be
+//       in the kubernetes.io/ namespace.
+func IsCustomNamespacedResourceName(name v1.ResourceName) bool {
```

Comment:
This is just the inverse of the IsDefaultNamespaceResource() function in pkg/api/helper/helpers.go. Can you at least add a comment to both of them so that someone in the future who changes them will ensure they stay in sync?